template.txt

```diff
@@ -15,6 +15,7 @@ stanford_path_to_models_jar = "./stanford-corenlp-full-2018-10-05/stanford-coren
 javahome = '*your_path_to_jdk_bin_java.exe*'
 
 #DOCKER variables
+    DOCKER = "Y"
```

Comment:
I don't see the corresponding change in `config.py`. There too, it should check for DOCKER = Y to load these values instead of the current implementation of any value. Since, now DOCKER = N specifies a normal setup.
Also, the default value here must be DOCKER = N as discussed for normal setup.